tools/Microsoft.AspNetCore.Server.Kestrel.GeneratedCode/KnownHeaders.cs

```diff
@@ -371,7 +371,9 @@ protected override bool RemoveFast(string key)
             return MaybeUnknown?.Remove(key) ?? false;
         }}
         protected override void ClearFast()
-        {{
+        {{{(loop.ClassName != "FrameRequestHeaders" ? "" : Each(loop.Headers, header => $@"
+            if ({header.TestBit()}) _{header.Identifier} = default(StringValues);"))}
```

Comment:
Why not do this for response headers as well? I see the comment in the PR about those being controlled by the app and not the client, but it still seems nice to keep response header values out of higher generations when browsers keep connections open.